.github/workflows/export_database_views.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function create_view_file() {
+    local VIEW_NAME=$1
+
+    echo "View: $VIEW_NAME"
+    MOST_RECENT_MIGRATION=$(grep -rl --include=up.sql -E "CREATE (OR REPLACE )?VIEW (public.)?$VIEW_NAME" moped-database/migrations | sort -V | tail -n 1)
```

Comment:
this is rad. 😎 Do you think there is a chance this would fail if there are quotes around the schema or view name? This is only on my mind because I noticed that double quotes are valid around schema and view or table when I've been working on the data model stuff.

Reply:
What an insightful question!! I'm going to answer you out of my beliefs from experience but not having just run an experiment to verify my understanding. 

I think we're square here because the process is generating the view-driving-query out of the postgres server's internal representation of the query.  I believe that the PG server outputs SQL in a very specific linted format, which doesn't use quotes. I've found that it refines queries in a whole bunch of ways, almost always adding specificity where humans might have let the query-engine infer things. 

I'll do a quick test tomorrow to firm up my understanding. Thanks again for this very thoughtful question.

Reply:
Cool, yeah, I was missing the step that it is coming directly from the DB. I only went as far as seeing what formatting would output from the Hasura console. Thanks for checking it out!

Reply:
This was addressed here, which also got the exports up to date with the latest changes as well: https://github.com/frankhereford/atd-moped/commit/87675e72da3353c1a3040280e6c1296ec6e83e3c